internal/database/database_test.go

```diff
@@ -0,0 +1,69 @@
+package database
+
+import (
+	"github.com/autobrr/autobrr/internal/domain"
+	"github.com/autobrr/autobrr/internal/logger"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"testing"
+)
+
+func getDbs() []string {
+	return []string{"sqlite", "postgres"}
+}
+
+func SetupDatabase(t *testing.T, dbType string) *DB {
+	err := os.Setenv("IS_TEST_ENV", "true")
+	if err != nil {
+		t.Fatalf("Could not set env variable: %v", err)
+		return nil
+	}
+
+	cfg := &domain.Config{
+		LogLevel:         "INFO",
+		DatabaseType:     dbType,
+		PostgresHost:     "localhost",
+		PostgresPort:     5437,
+		PostgresDatabase: "autobrr",
+		PostgresUser:     "testdb",
+		PostgresPass:     "testdb",
+	}
+
+	// Init a new logger
+	log := logger.New(cfg)
+
+	// Initialize a new DB connection
+	db, err := NewDB(cfg, log)
+	if err != nil {
+		t.Fatalf("Could not create database: %v", err)
+	}
+
+	// Open the database connection
+	if err := db.Open(); err != nil {
+		t.Fatalf("Could not open db connection: %v", err)
+	}
+
+	return db
+}
+
+func SetupLogger() logger.Logger {
```

Comment:
Same as above, does not need to be exported and rename to `setupLoggerForTest` makes it a bit more descriptive.

Reply:
:+1: